package-lock.json

```diff
@@ -17339,7 +17339,7 @@
       "from": "github:mozillareality/three.js#hubs/master"
     },
     "three-ammo": {
-      "version": "github:infinitelee/three-ammo#c613987b7ae126fdb570f10477399a5cac4d70f6",
+      "version": "github:infinitelee/three-ammo#fe57353506b79627a5b2e9b0feba4a2fe52b745c",
```

Comment:
https://github.com/InfiniteLee/three-ammo/commit/fe57353506b79627a5b2e9b0feba4a2fe52b745c#diff-f0c01727e5628483090afb5bc8c36be6L377

This is the change that affects the behavior we can see